### DISCUSSION TITLE
V3 Multitenancy limit users in same panel to create new tenants

### DISCUSSION THREAD

**Discussion by wrtisans** [filamentphp/filament#7441] [Ideas]
my use case is 

each tenant has an admin (i've a root panel for me which handles all tenants) and each tenant only can register new tenant if is admin or has some permission (using spatie permissions and teams functionality built in with that package), is there a way to only allow the creation of new tenants or modify tenant profile with permissions without create new panel ?

**Comment by pboivin** [comment]
I think this should be possible with regular model policies. What have you been trying?
